### PR TITLE
feat: configure for sso integration

### PR DESCRIPTION
**Pull request by anthonywendt**
## Description
Updates and cleanup to allow sso integration
...

## Related Issue

Relates to https://github.com/defenseunicorns/uds-bundle-software-factory-nutanix/issues/47

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Other (security config, docs update, etc)

## Checklist before merging

- [x] Test, docs, adr added or updated as needed
- [x] [Contributor Guide Steps](https://github.com/defenseunicorns/uds-package-nexus/blob/main/CONTRIBUTING.md#developer-workflow) followed

### CODE REVIEW

File: src/dev-secrets/zarf.yaml [hunk 1]
```diff
@@ -11,8 +11,13 @@ components:
     actions:
       onDeploy:
         before:
-          - cmd: kubectl get secret -n postgres nexus.pg-cluster.credentials.postgresql.acid.zalan.do --template={{.data.password}} | base64 -d
+          - cmd: kubectl get secret -n nexus nexus.nexus.pg-cluster.credentials.postgresql.acid.zalan.do --template={{.data.password}} | base64 -d
```
Comment (blancharda) [inline src/dev-secrets/zarf.yaml:14]: Should this be an internal reference (`./zarf tools kubectl`)?

File: src/dev-secrets/zarf.yaml [hunk 2]
```diff
@@ -11,8 +11,13 @@ components:
...
             mute: true
             setVariables:
               - name: NEXUS_DB_PASSWORD
                 sensitive: true
+          - cmd: kubectl get secret -n nexus nexus.nexus.pg-cluster.credentials.postgresql.acid.zalan.do --template={{.data.username}} | base64 -d
```
Comment (blancharda) [inline src/dev-secrets/zarf.yaml:19]: Same

File: chart/values.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+domain: "###ZARF_VAR_DOMAIN###"
+sso:
+  enabled: true
```
Comment (blancharda) [inline chart/values.yaml:3]: Do we want the default here to be true?

**Review by blancharda — COMMENTED** [review]
Minor nits -- overall looks good!

File: chart/values.yaml (hunk 3 above)
Comment (anthonywendt) [reply chart/values.yaml:3]: Yes. Going to clean this up

**Review by blancharda — COMMENTED** [review]
LGTM!

**Pull request merged**